openqabot/utils.py

```diff
@@ -21,6 +22,20 @@ def create_logger(name: str) -> logging.Logger:
     return log
 
 
+def get_yml_list(path: Path) -> List[Path]:
+    """
+    Create a list of YML filesnames from a folder
+    or from a single file path.
+    """
+    yml_list = []
+    extension = "*.yml"
+    if path.is_file() and path.match(extension):
```

Comment:
Maybe it is just fine but a yaml file can have either `yml` or `yaml`. The second is not gonna ever match.

Reply:
seems by design https://github.com/openSUSE/qem-bot/blob/master/doc/config.md?plain=1#L4

```
All configuration files must have `.yml` extension.
```

I like the idea to extend it, eventually in a different PR.